modules/display.js

```diff
@@ -0,0 +1,9 @@
+export default function displayBook(book) {
```

Comment:
- [ ] I will be happy if we can start making use of [ES6 arrow functions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions)